src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -260,12 +260,40 @@ def _encode_prompt(
         prompt_embeds: Optional[torch.FloatTensor] = None,
         negative_prompt_embeds: Optional[torch.FloatTensor] = None,
         lora_scale: Optional[float] = None,
+        concat_prompt_embeds=True,
+    ):
+        deprecation_message = "`_encode_prompt()` is now deprecated and it will be removed in a future version. Use `encode_prompt()` instead."
```

Comment:
```suggestion
        deprecation_message = "`_encode_prompt()` is deprecated and it will be removed in a future version. Use `encode_prompt()` instead and be aware that the output format changed from a concatenated tensor to a tulpe."
```